rustls/src/error.rs

```diff
@@ -765,7 +765,7 @@ impl fmt::Display for CertificateError {
                 Ok(())
             }
 
-            other => write!(f, "{other:?}"),
+            other => write!(f, "{other}"),
```

Comment:
This needs to be different in some way. Either we need to write a full set if display handling for each variant, and then call `Display` on the value contained in the `Other(_)` variant, or we need to pick out the `Other(_)` variant and keep calling `Debug` on the catch-all case.

At the moment it unconditionally recurses forever.

Reply:
sorry about that, I didn’t look through it carefully (took it a bit lightly since it seemed like a trivial change)